src/main/webapp/css/style-userpage.css

```diff
@@ -0,0 +1,55 @@
+#recipe-modal {
+  background-color: rgba(0,0,0,0.7); /* Black w/ opacity */
+  display: none; /* Hidden by default */
+  height: 100%; /* Full height */
+  left: 0;
+  overflow: auto; /* Enable scroll if needed */
+  padding-top: 100px;
+  position: fixed; /* Stay in place */
+  top: 0;
+  width: 100%; /* Full width */
+  z-index: 1; /* Sit on top */
+}
+#recipe-form {
```

Comment:
Can you include a line between your CSS rules? Ref: [CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation)